torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -13,6 +14,50 @@
 )
 
 
+###############################################################################
+# Kernel Side Table
+
+
+# We cannot put Triton Kernels into the FX graph as the graph nodes
+# do not support arbitrary functions.
+# Use a side table.
+# We use two dicts so that fetching both the kernel and id are O(1)
+class KernelSideTable:
+    id_to_kernel: Dict[int, Any] = dict()
```

Comment:
typing this code requires importing triton which comes with its own challenges, perhaps we should just create a fake triton import at some point to define some of the top level types